pkg/publisher/publisher.go

```diff
@@ -136,10 +133,11 @@ func New(ctx context.Context, region string, clusterID string, log logger.Logger
 	}, nil
 }
 
-// Start is used to setup the monitor loop
-func (p *cloudWatchPublisher) Start() {
-	p.log.Info("Starting monitor loop for CloudWatch publisher")
-	p.monitor(defaultInterval)
+// Start is used to set up the monitor loop
+func (p *cloudWatchPublisher) Start(publishInterval int) {
+	p.log.Infof("Starting monitor loop for CloudWatch publisher with interval (%d)", publishInterval)
+	var publishIntervalDuration = time.Second * time.Duration(publishInterval)
```

Comment:
nit: no need to cast int to `time.Duration`. And can you also update the log to something like "with push interval of %d seconds"?

Reply:
Completed in the latest commit